fixtures/eurolite/led-b-40.json

```diff
@@ -0,0 +1,425 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED B-40",
+  "categories": ["Laser", "Flower", "Effect", "Color Changer"],
+  "meta": {
+    "authors": ["Charlélie DESMET", "Flo Edelmann"],
+    "createDate": "2019-09-23",
+    "lastModifyDate": "2019-09-23"
+  },
+  "comment": "Laser Beam Effect",
```

Comment:
Maybe better explain this effect, i.e. that there are three different RGBW LEDs inside the sphere? Otherwise, the matrix structure looks quite odd compared to the product image.